### DISCUSSION TITLE
how to add labels in our issue like gsoc -extend or hactobufest

### DISCUSSION THREAD

**Discussion by Ayush-23479** [GarimaSingh0109/Resum-Resume#571] [Q&A]
I am facing this issue that how can we add lablels in our issue like gsoc-extend or hactobufest

**Comment by APURVA122** [comment]
TO ADD LABELS: 
Comment on the Issue: You can leave a comment on the issue which you would like to add label, politely asking if they could add the specific label(s) you need.

Tag Maintainers:  you can tag the admin  in your comment using @username to bring it to their attention.

Provide Context: It can be helpful to explain why you think the label should be added, especially if it relates to a specific event or initiative like Hacktoberfest or GSoC.

Being clear and polite in your request usually helps get a positive response!
